.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,41 @@
+<!-- Read comments, before committing pull request read checklist again
+
+# Checklist:
+
+- I have performed a self-review of my own code,
+- I have commented my code, particularly in hard-to-understand areas,
+- Title include "WIP" if work is in progress.
+
+-->
+
+Resolves #01 <!-- issue number -->
```

Comment:
Lets update this to `Resolves #<ISSUE_NUMBER>` so that by mistake no one links the PR to first issue by mistake.

Reply:
I will fix this by the end of the day.